src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,27 @@
+package duke.command;
+
+import duke.exception.DukeException;
+import duke.storage.Storage;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+public abstract class Command {
+
+    /**
+     * Executes based on the command with the arguments provided.
+     *
+     * @param tasks Task List that store all current task.
+     * @param ui Ui object to interact with user.
+     * @param storage Storage that store tasks in hard-drive.
+     * @return Expected message to be sent to user.
+     * @throws DukeException If arguments enters has error.
+     */
+    public abstract String execute(TaskList tasks, Ui ui, Storage storage) throws DukeException;
+
+    /**
```

Comment:
Not 100% sure about this: since this is an abstract method, it doesn't have a return value right? Are you sure @return should be included in the javadoc?